packages/loader/container-loader/api-report/container-loader.legacy.alpha.api.md

```diff
@@ -48,6 +51,15 @@ export interface IFluidModuleWithDetails {
     module: IFluidModule;
 }
 
+// @alpha
+export interface ILoaderCreateDetachedContainerProps extends ILoaderProps {
```

Comment:
```suggestion
export interface ICreateDetachedContainerProps extends ILoaderProps {
```

Reply:
let's remove ILoader from these, as we will probably want to deprecate that as well eventually

Reply:
moved to code rather than docs